public/js/extra.js

```diff
@@ -1036,6 +1038,33 @@ md.renderer.rules.fence = (tokens, idx, options, env, self) => {
   return `<pre><code${self.renderAttrs(token)}>${highlighted}</code></pre>\n`
 }
 
+const makePlantumlURL = (umlCode) => {
+  let format = 'svg'
+  let code = plantumlEncoder.encode(umlCode)
+  return `${plantumlServer}/${format}/${code}`
```

Comment:
Access explicitly from window can remind us where it's from, and the eslint global rule can also be removed after this changed.

```suggestion
  return `${window.plantumlServer}/${format}/${code}`
```

Reply:
@chulkilee could you consider this?

Reply:
I don't know how that config is injected to windows. It seems like that (global, as the top eslint rule says).

However, as there is already `serverUrl` - I was confused - that why `plantumlServer` is special here.

Probably config should be exposed as `config` - and we should use `window.config.xxx`, instead of `window.xxx`. I believe that should be a separate PR.

Reply:
Ok I see 👍, We can do this refactor later, providing cleaner config interface. Let's merge this PR @jackycute !

Reply:
BTW, global accessible constants is loaded via script tag https://github.com/hackmdio/codimd/blob/6bce9ac5bf4d98f65f55621ea5073781328ba2a1/public/views/includes/scripts.ejs#L1

which generated in https://github.com/hackmdio/codimd/blob/1d7895c8693e30e4291576d293a9998db6d461c0/lib/web/statusRouter.js#L94-L110